lib/iscsi-command.c

```diff
@@ -319,6 +319,14 @@ static void parse_sense_spec(struct scsi_sense *sense, const uint8_t inf[3])
 	sense->field_pointer = scsi_get_uint16(&inf[1]);
 }
 
+/* Information field length differs between Descriptor and Fixed types */
```

Comment:
Since the body of this function is only two lines long, please inline this function.